docs/Golden-Gate/Bridge-Walkthrough.rst

```diff
@@ -0,0 +1,144 @@
+Bridge Walkthrough
+======================
+In this section, we'll walkthrough a simple Target-to-Host bridge provided with FireSim
+to demonstrate how to integrate your own. This includes the UARTBridge, which
+uses host-MMIO to model a UART device -- this is what you're using when you
+attach to a FireSim simulator's console.
+
+Reading the Bridges section is a prerequisite to reading these sections.
+
+UART Bridge (Host-MMIO)
+-----------------------
+
+Source code for the UART Bridge lives in the following memory locations:
+
+.. code-block:: text
+
+    sim/
+    ├-firesim-lib/src/main/
+    │                 ├-scala/bridges/UARTBridge.scala # Target-Side Bridge and BridgeModule Definitions
+    │                 ├-cc/brides/uart.cc # Bridge Driver source
+    │                 └-cc/brides/uart.h  # Bridge Driver header
+    ├-src/main/cc/firesim/firesim_top.cc  # Driver instantiation in the main simulation driver
+    └-src/main/makefrag/firesim/Makefrag  # Build system modifications to compile Bridge Driver code
+
+UART Bridge Target-Side Chisel
```

Comment:
To be consistent with the high-level documentation page (which you expect people to read before getting to this page), maybe call it "UART Target-Side Bridge"? (rather than Target-Side Chisel)